Summary_test.go

```diff
@@ -1,84 +0,0 @@
-package main
-
-import (
-	"net/http"
-	"os"
-
-	"github.com/guzzlerio/corcel/serialisation/yaml"
-	"github.com/guzzlerio/corcel/statistics"
-	"github.com/guzzlerio/corcel/utils"
-	"github.com/guzzlerio/rizo"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("Run Summary", func() {
```

Comment:
No you are correct I have neglected to cover the actual set of shell tests which we need too